### PR TITLE
Update Anchor stories to new Format

### PR DESCRIPTION
**Pull request by leossantiago**
#### What does this PR do?

- Update Anchor stories to Controls/Anchor.
- Rename `Colors` to `Color` to match other components' stories with the same name.

#### Where should the reviewer start?

`components/Anchor/stories/Anchor.stories.js`

#### What testing has been done on this PR?

Storybook

#### How should this be manually tested?

Storybook

#### What are the relevant issues?

#4651 

#### Screenshots (if appropriate)

image

#### Do the grommet docs need to be updated?

No.

#### Is this change backwards compatible or is it a breaking change?

Backwards compatible.

### CODE REVIEW

File: src/js/components/Anchor/stories/Default.js [hunk 1]
```diff
@@ -14,6 +13,4 @@ const Default = () => {
   );
 };
 
-storiesOf('Anchor', module).add('Default', () => <Default />, {
-  chromatic: { disable: true },
-});
+export const Default = () => <DefaultAnchor />;
```
Comment (jcfilben) [inline src/js/components/Anchor/stories/Default.js:16]: Chromatic should still be disabled on the story. To see an example of this look at Accordion/stories/Simple.js in this PR https://github.com/grommet/grommet/pull/4654

Comment (leossantiago) [reply src/js/components/Anchor/stories/Default.js]: Fixed :)

**Review by jcfilben — APPROVED** [review]
PR looks good, I'm approving. However, I realized we have two versions of this PR. Other version is https://github.com/grommet/grommet/pull/4663. This one (4662) was filed first but they both do the same thing so either one can be merged

**Comment by ShimiSun** [comment]
Thank you for your contribution!  🎉

**Pull request merged**